rpc/README.md

```diff
@@ -1659,12 +1659,12 @@ Response
         "genesis_hash": "0x7978ec7ce5b507cfb52e149e36b1a23f6062ed150503c85bbf825da3599095ed",
         "hardfork_features": [
             { "rfc": "0028", "epoch_number": "0x1526" },
-            { "rfc": "0029", "epoch_number": "0x0" },
-            { "rfc": "0030", "epoch_number": "0x0" },
-            { "rfc": "0031", "epoch_number": "0x0" },
-            { "rfc": "0032", "epoch_number": "0x0" },
-            { "rfc": "0036", "epoch_number": "0x0" },
-            { "rfc": "0038", "epoch_number": "0x0" },
+            { "rfc": "0029", "epoch_number": "0x1526" },
+            { "rfc": "0030", "epoch_number": "0x1526" },
+            { "rfc": "0031", "epoch_number": "0x1526" },
+            { "rfc": "0032", "epoch_number": "0x1526" },
+            { "rfc": "0036", "epoch_number": "0x1526" },
+            { "rfc": "0038", "epoch_number": "0x1526" },
```

Comment:
Here as I understand it, it should just restore vm version selection related, not all.